src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -9,7 +9,9 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PlatformTarget>AnyCPU</PlatformTarget>
-	<DefineConstants>$(DefineConstants);PROJECTK</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetsWindows)' != 'true' ">
+    <DefineConstants>$(DefineConstants);MANAGED_SNI</DefineConstants>
```

Comment:
It's actually the opposite. There are a lot of features in SqlClient like LocalDB, Integrated Auth, and Named Pipes that are all Windows specific. So we're currently using the native codebase to get around these issues until we create managed solutions.